source/common/http/header_map_impl.cc

```diff
@@ -346,6 +359,13 @@ void HeaderMapImpl::addReferenceKey(const LowerCaseString& key, const std::strin
 }
 
 void HeaderMapImpl::addCopy(const LowerCaseString& key, uint64_t value) {
+  auto* entry = getExistingInline(key.get().c_str());
+  if (entry != nullptr) {
+    char buf[32];
+    StringUtil::itoa(buf, sizeof(buf), value);
+    appendToHeader(entry->value(), buf);
```

Comment:
I just noticed this, but we are creating an `std::string` in this path. Can we make this not create a string here also for perf reasons?